modules/dns/dns_api.rb

```diff
@@ -29,6 +29,8 @@ class Api < ::Sinatra::Base
         when 'CNAME'
           validate_dns_name!(value)
           server.create_cname_record(fqdn, value)
+        when 'SSHFP'
```

Comment:
Can you verify the value? It doesn't need exact but at least verify there are (exactly) 3 values if you split it.